src/app/src/components/Dashboard.jsx

```diff
@@ -27,9 +40,38 @@ function Dashboard({
         );
     }
 
+    const linkSection = (
+        <div style={dashboardStyles.linkSectionStyles}>
+            <Link to={dashboardListsRoute}>
+                View Contributor Lists
+            </Link>
+            <FeatureFlag flag={CLAIM_A_FACILITY}>
+                <Link to={dashboardClaimsRoute}>
+                    View Facility Claims
+                </Link>
+            </FeatureFlag>
+        </div>
+    );
+
     return (
-        <AppGrid title="dashboard">
-            Dashboard
+        <AppGrid title="Dashboard">
+            <Switch>
+                <Route path={dashboardListsRoute} component={DashboardLists} />
+                <Route
+                    path={dashboardClaimsRoute}
+                    render={
+                        () => (
+                            <FeatureFlag
+                                flag={CLAIM_A_FACILITY}
+                                alternative={linkSection}
+                            >
+                                <Route component={DashboardClaims} />
```

Comment:
The nesting here is to enable using the `FeatureFlag` component while keeping react router props in DashboardClaims